OemPkg/SecureBootRecovery/helper.py

```diff
@@ -0,0 +1,13 @@
+# Take the contents of the Payload folder and convert it into a C Header file
+
+from edk2toollib.utility_functions import export_c_type_array
+
+with open("Payload/dbUpdate.bin", "rb") as payload_fs, open("./RecoveryPayload.h", "w") as output_fs:
+    output_fs.write("#ifndef _RECOVERY_PAYLOAD_H_\n")

```

Comment:
We require CRLF for line endings. Since this is generating source code that could update a checked in source file, it should use those line endings.

Reply:
TIL when opening a file in `text` mode it converts `\n` to os.linesep 

> When writing output to the stream, if newline is None, any '\n' characters written are translated to the system default line separator, [os.linesep](https://docs.python.org/3/library/os.html#os.linesep). If newline is '' or '\n', no translation takes place. If newline is any of the other legal values, any '\n' characters written are translated to the given string.
https://docs.python.org/3/library/functions.html#open

Reply:
> TIL when opening a file in `text` mode it converts `\n` to os.linesep

Yeah, that helps if someone runs the script on Windows. Explicit CRLF allows the script to generate consistent line endings on Windows and Linux and keep those in the normal Linux configuration for our codebases where git core.autocrlf = false.